actors/miner/tests/util.rs

```diff
@@ -2220,7 +2140,16 @@ impl ActorHarness {
                 }
             }
 
-            self.advance_deadline(rt, CronConfig::empty());
+            let unvested: TokenAmount = unvested_vesting_funds(rt, &state);
+            let available = rt.get_balance() + unvested.clone() - &state.initial_pledge;
+            let burnt_funds =
+                std::cmp::max(TokenAmount::zero(), std::cmp::min(daily_fee.clone(), available));
+            let pledge_delta = std::cmp::min(unvested, daily_fee).neg();
+
+            //
+            let cfg = CronConfig { burnt_funds, pledge_delta, ..Default::default() };
+            self.advance_deadline(rt, cfg);
```

Comment:
I don't love that we're still doing lots of calculation in here, but `advance_and_submit_posts()` is such a common activity that I think the trade-off is worth it for the generic case where we're not expecting things to go wrong.